react-basic-practice/src/components/CustomerView/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { Box, FormLabel, Grid, GridItem, Text } from "@chakra-ui/react";
+import { ICustomer } from "@types";
+import TableFinance from "../TableFinance";
+import TableStatus from "../TableStatus";
+
+interface ViewCustomerProps {
+  data: ICustomer;
+}
+
+const FormView: React.FC<ViewCustomerProps> = ({ data }) => {
```

Comment:
Pls rename same as folder name. Pls update for CustomerForm too

Reply:
i done